var/spack/repos/builtin/packages/py-dask-mpi/package.py

```diff
@@ -23,6 +23,7 @@ class PyDaskMpi(PythonPackage):
     depends_on("py-dask@2.2:", when="@:2.21.0", type=("build", "run"))
     depends_on("py-dask@2.19:", when="@2022.4.0:", type=("build", "run"))
     depends_on("py-distributed@2.19:", when="@2022.4.0:", type=("build", "run"))
-    depends_on("py-jupyter-server-proxy", type=("build", "run"))
     depends_on("py-mpi4py", type=("build", "run"))
     depends_on("py-mpi4py@3.0.3:", when="@2022.4.0:", type=("build", "run"))
+
```

Comment:
Can you also add a comment linking to your PR that removed this? Could even add that PR as a URL patch. Also, did you check that the patch applies to all versions? You should also add an upper bound since it won't be necessary in the next release.

Reply:
> Can you also add a comment linking to your PR that removed this?

Done.

> You should also add an upper bound since it won't be necessary in the next release.

And done.

Thanks for bearing with me.